dffml/service/dev.py

```diff
@@ -1071,3 +1201,4 @@ class Develop(CMD):
     setuppy = SetupPy
     bump = Bump
     ci = CI
+    lintcommits = CommitLint
```

Comment:
Let's make sure we keep this the same only lowercase

Reply:
Made it `LintCommits`